src/shared/nested-dropdown-list/nested-dropdown-list.tsx

```diff
@@ -156,6 +156,7 @@ export const NestedDropdownList = <V1, V2, V3>({
 
     const handleSelect = (item: CombinedFormattedOptionProps<V1, V2, V3>) => {
         const { label, keyPath, value } = item;
+        updateSelectedState(keyPath);
```

Comment:
does nested dropdown list need to track the selected state? `input-nested-multi-select` would have passed down an updated `selectedKeyPaths` which should trigger the relevant useEffect

Reply:
yes. my bad. this seems like pointless, the dropdown will closed after selected.